wp-content/themes/eighshift-task/src/blocks/custom/posts/assets/load-more.js

```diff
@@ -0,0 +1,61 @@
+import '@babel/polyfill';
+
+export class LoadMore {
```

Comment:
try to make this method as reusable as possible, and always think of that when you are coading